tasks/clean.js

```diff
@@ -1,10 +1,8 @@
-import del from 'del';
+import { deleteSync } from 'del';
 import gulp from 'gulp';
 
 function clean(resolve) {
-  del(['docs/*'], {
-    dot: true
-  });
+  deleteSync(['docs/*']);
```

Comment:
`del` does not exist with the current version, so I updated it to the corresponding command.

Reply:
https://www.npmjs.com/package/del

Reply:
Seems reasonable!